i14262: Edited the 'intro to che, installing chectl, & running che lo… (#790)
codenvy/che-docs: src/main/pages/che-7/overview/assembly_running-che-locally.adoc

[id="running-che-locally_{context}"]
= Running Che locally

This section contains instructions for deploying and running Eclipse Che locally, on a personal workstation.

.Prerequisites

To run and manage Che:

* A Kubernetes or OpenShift cluster to deploy Che on
* The link:https://github.com/che-incubator/chectl[`chectl`] command-line tool for managing a Che server and its development workspaces. See link:{site-baseurl}che-7/installing-the-chectl-management-tool/[Installing the `chectl` management tool].

Eclipse Che is available in two modes:

* *Single-user*: Non-authenticated Che, lighter and suited for personal desktop environments
* *Multi-user*: Authenticted Che, suited for the cloud, for organizations and developer teams

// TODO: See link:single-multi-user.html[Single and Multi-User Che] to learn more.

> Instructions in this section focus on deploying and running Che in the *single-user* mode.

Reviewer: ```suggestion
This section describes how to deploy and run Che in *single-user* mode.
```

=== Using Minishift to set up OpenShift

> . Install the latest version of link:https://docs.okd.io/latest/install/index.html[Minishift] and link:https://docs.okd.io/latest/minishift/command-ref/minishift_oc-env.html[set the path of the `oc` binary].

Reviewer: neither of the links work

Author: @rkratky , I changed the Minishift installation link and it works for me. But, the one for setting up oc binary one does not. I am looking for the correct one.

Author: @rkratky , strangely, the link "https://docs.okd.io/latest/minishift/command-ref/minishift_oc-env.html" is now working.
